src/i18n/messages/en.messages.json

```diff
@@ -93,10 +93,10 @@
       "addToCart": "ADD TO CART"
     },
     "spellcheck": {
-      "message": "No results found for <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. We show you results for"
+      "message": "No results found for <span class=\"x-font-bold x-text-auxiliary-50\">'{query}'</span>. We show you results for"
     },
     "noResults": {
-      "message": "<span>Sorry but there are no results for <span class=\"x-text x-font-color--neutral-10 x-text--bold x-font-size--05\">\"{query}\"</span></span>You may be interested in these products"
+      "message": "<span>Sorry but there are no results for <span class=\"x-font-bold\">\"{query}\"</span></span>You may be interested in these products"
```

Comment:
```suggestion
      "message": "<span>Sorry but there are no results for <span class=\"x-title3\">\"{query}\"</span></span>You may be interested in these products"
```

Reply:
I'm seeing `title-1` in the table for the no results message component